shared/metrics/src/test/java/io/pravega/shared/metrics/MetricsProviderTest.java

```diff
@@ -107,8 +107,8 @@ public void testGauge() {
         for (int i = 1; i < 10; i++) {
             value.set(i);
             dynamicLogger.reportGaugeValue("dynamicGauge", i);
-            assertEquals(i, MetricsProvider.METRIC_REGISTRY.getGauges().get("testGauge").getValue());
-            assertEquals(i, MetricsProvider.METRIC_REGISTRY.getGauges().get("DYNAMIC.dynamicGauge.Gauge").getValue());
+            assertEquals(i, MetricsProvider.METRIC_REGISTRY.getGauges().get("pravega.testStatsLogger.testGauge").getValue());
+            assertEquals(i, MetricsProvider.METRIC_REGISTRY.getGauges().get("pravega.dynamicGauge.Gauge").getValue());
```

Comment:
Yes. See all names in issue comment: https://github.com/pravega/pravega/issues/1370#issuecomment-307305905